schemes/rayleigh_friction/rayleigh_friction.meta

```diff
@@ -0,0 +1,126 @@
+[ccpp-table-properties]
+  name = rayleigh_friction
+  type = scheme
+
+[ccpp-arg-table]
+  name  = rayleigh_friction_init
+  type  = scheme
+[ pver ]
+  standard_name = vertical_layer_dimension
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ raytau0_nl ]
+  standard_name = rayleigh_friction_model_top_decay_time
```

Comment:
I'll leave this question for @nusbaume during his review:

how important is it for these scheme-specific / namelist standard names to follow the official rules? I think this should technically be `model_to_decay_time_for_rayleigh_friction`, but I guess I'm not sure how picky we want to be.

Reply:
I changed this one, and "rayleigh_friction_center_level" to "center_vertical_layer_for_rayleigh_friction"